slim/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN set -x \
     && cd / && rm -rf /znc-src
 
 COPY docker-entrypoint.sh /
-
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
I will try build the image again on my rpi and my laptop to see if this occurs again.